iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/simulator/TestDigitalTwinDevice.java

```diff
@@ -0,0 +1,63 @@
+package tests.integration.com.microsoft.azure.sdk.iot.digitaltwin.simulator;
+
+import com.microsoft.azure.sdk.iot.device.ClientOptions;
+import com.microsoft.azure.sdk.iot.device.DeviceClient;
+import com.microsoft.azure.sdk.iot.device.IotHubClientProtocol;
+import com.microsoft.azure.sdk.iot.service.Device;
+import com.microsoft.azure.sdk.iot.service.RegistryManager;
+import com.microsoft.azure.sdk.iot.service.auth.AuthenticationType;
+import com.microsoft.azure.sdk.iot.service.exceptions.IotHubException;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import tests.integration.com.microsoft.azure.sdk.iot.digitaltwin.helpers.E2ETestConstants;
+import tests.integration.com.microsoft.azure.sdk.iot.helpers.Tools;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+@Slf4j
+@Getter
+public class TestDigitalTwinDevice {
+    private static final String IOT_HUB_CONNECTION_STRING = Tools.retrieveEnvironmentVariableValue(E2ETestConstants.IOT_HUB_CONNECTION_STRING_ENV_VAR_NAME);
```

Comment:
nit - Everywhere else (samples) we are using this - IOTHUB_CONNECTON_STRING so we can follow that convention.

Reply:
Done